stories/examples/Map.js

```diff
@@ -2,82 +2,69 @@ import React, { Component } from 'react'
-        "type": "FeatureCollection",
-        "features": [{
-          "type": "Feature",
-          "geometry": {
-            "type": "Point",
-            "coordinates": [-84.424, 33.979]
-          },
-          "properties": {
-            "title": "Point 1",
-            "marker-symbol": "secondary_marker"
-          }
-        }, {
-          "type": "Feature",
-          "geometry": {
-            "type": "Point",
-            "coordinates": [-84.534, 33.676]
-          },
-          "properties": {
-            "title": "Point 2",
-            "marker-color": "#ff00ff",
-            "marker-symbol": "secondary_marker"
-          }
-        }, {
-          "type": "Feature",
-          "geometry": {
-            "type": "Point",
-            "coordinates": [-84.234, 33.776]
-          },
-          "properties": {
-            "title": "Point 3",
-            "marker-color": "#ff00ff",
-            "marker-symbol": "secondary_marker"
-          }
-        }]
+  const mapSourceData = {
+    "type": "FeatureCollection",
+    "features": [{
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [-84.424, 33.979]
+      },
+      "properties": {
+        "title": "Point 1",
+        "marker-symbol": "secondary_marker"
       }
-    }
-  }
-]
-
-const mapLayerData = [
-  {
-    layer: {
-      "id": "markers-default",
-      "source": "markers",
-      "type": "symbol",
-      "layout": {
-        "icon-image": "{marker-symbol}",
-        "text-field": "{title}",
-        "text-font": ["Open Sans Semibold", "Arial Unicode MS Bold"],
-        "text-offset": [0, 0.6],
-        "text-anchor": "top"
+    }, {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [-84.534, 33.676]
+      },
+      "properties": {
+        "title": "Point 2",
+        "marker-color": "#ff00ff",
+        "marker-symbol": "secondary_marker"
       }
-    }
-  },
-  {
-    layer: {
-      "id": "markers-active",
-      "source": "markers",
-      "type": "symbol",
-      "layout": {
-        "icon-image": "default_marker",
-        "text-field": "{title}",
-        "text-font": ["Open Sans Semibold", "Arial Unicode MS Bold"],
-        "text-offset": [0, 0.6],
-        "text-anchor": "top"
+    }, {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [-84.234, 33.776]
       },
-      "filter": ["==", "title", ""]
+      "properties": {
+        "title": "Point 3",
+        "marker-color": "#ff00ff",
+        "marker-symbol": "secondary_marker"
+      }
+    }]
+  }
+
+  const markersDefaultData = {
+    "id": "markers-default",
+    "source": "markers",
+    "type": "symbol",
+    "layout": {
+      "icon-image": "{marker-symbol}",
+      "text-field": "{title}",
+      "text-font": ["Open Sans Semibold", "Arial Unicode MS Bold"],
+      "text-offset": [0, 0.6],
+      "text-anchor": "top"
     }
   }
-]
+
+  const markersActiveData = {
```

Comment:
was storybook ever displaying this properly? I am not seeing this wired up in the example

Reply:
@maityarkodeb showed me it is just faint.